### PR TITLE
Added version flag to CLI

### PR DESCRIPTION
**Pull request by rewati-arista**
# Please include a summary of the changes

* vane_cli.py: added version flag

# Include the Issue number and link

https://github.com/aristanetworks/vane/issues/540

# Type of change

- - [ ] Bug fix (non-breaking change which fixes an issue)
- - [X] New feature (non-breaking change which adds functionality)
- - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- - [ ] This change requires a documentation update
- - [ ] Other (please specify)

# Effort required on reviewers end

- - [X] Easy
- - [ ] Medium
- - [ ] Hard 

# How Has This Been Tested?
    
## New feature

    
Screenshot 2023-09-27 at 12 42 09 PM

# CI pipeline result

- - [X] Pass
- - [ ] Fail

    
# Additional comments

To be tested with setup.py format as well

### CODE REVIEW

File: vane/vane_cli.py [hunk 1]
```diff
@@ -37,6 +37,7 @@
 from io import StringIO
 from contextlib import redirect_stdout
 from datetime import datetime
+from importlib import metadata
```
Comment (kgrozis-arista) [inline vane/vane_cli.py:40]: Is importlib part of standard python distribution or do we need to add to pyproject.toml?

Comment (rewati-arista) [reply vane/vane_cli.py:40]: It is, here is the documentation https://docs.python.org/3.9/library/importlib.metadata.html :)

Comment (kgrozis-arista) [reply vane/vane_cli.py:40]: Thanks!

**Review by kgrozis-arista — APPROVED** [review]

**Review by grybak-arista — APPROVED** [review]
Looks good!

**Pull request merged**